tyk-single-dc/values.yaml

```diff
@@ -663,6 +686,14 @@ tyk-enterprise-portal:
   securityContext:
     runAsUser: 1000
     fsGroup: 2000
+  containerSecurityContext:
```

Comment:
Can you add a comment above this too like you did with others?

Reply:
Yeah. That we need to.
I thought we can start with your change. But we can handle it in separate PR.